source/new_project_process.adoc

```diff
@@ -0,0 +1,62 @@
+= Adding a new module/component/project to github
+
+This document outlines how to create a new repository for inclusion in JBoss Tools or JBDS. 
+
+== Consensus on Requirements
+
+First, write and submit a proposal to mailto:max.andersen@redhat.com[max.andersen@redhat.com] for review, explaining why your proposed project should stand alone, rather than being part of an existing JBoss Tools project. Alternatively, you can send the proposal to mailto:jbosstools-dev@lists.jboss.org[jbosstools-dev@lists.jboss.org] for wider distribution and discussion. 
```

Comment:
revert this - it should go to jbosstools-dev@lists.jboss.org first, and only directly to manderse@redhat.com if for some reason it cannot be suggested publicly.